buildenv/jenkins/common/variables-functions.groovy

```diff
@@ -665,6 +684,10 @@ def set_test_targets() {
     }
     echo "TESTS:'${TESTS}'"
 
+    set_sdk_impl()
+}
+
+def set_sdk_impl() {
     // Set SDK Implementation, default to OpenJ9
     SDK_IMPL = buildspec.getScalarField("sdk_impl", 'all') ?: 'openj9'
     SDK_IMPL_SHORT = buildspec.getScalarField("sdk_impl_short", 'all') ?: 'j9'
```

Comment:
Is SDK_IMPL_SHORT used anywhere else?

Reply:
Test job generation

Reply:
Is it a different PR?

Reply:
This is existing functionality
https://github.com/eclipse/openj9/blob/master/buildenv/jenkins/common/pipeline-functions.groovy#L467
https://github.com/eclipse/openj9/blob/master/buildenv/jenkins/common/pipeline-functions.groovy#L598